src/viking.js

```diff
@@ -1,14 +1,71 @@
 // Soldier
-class Soldier {}
+class Soldier {
+  constructor(health, strength){
+    this.health = health
+    this.strength = strength
+  }
+  attack(){
+    return this.strength;
+  }
+  receiveDamage (damage){
+   this.health -= damage;
+  }
+}
 
 // Viking
-class Viking {}
+class Viking extends Soldier {
+  constructor(vikingName, vikingHealth, vikingStrength) {
+    super(vikingHealth, vikingStrength);
+    this.vikingName = vikingName;
+  }
+  receiveDamage(damage) {
+    this.health -= damage;
+    if (this.health > 0) {
+      return `${this.vikingName} has received ${damage} points of damage`;
+    } else if (this.health <= 0) {
+      return `${this.vikingName} has died in act of combat`;
+    }
+  }
+  battleCry() {
+    return 'Odin Owns You All!';
+  }
+}
 
 // Saxon
-class Saxon {}
+class Saxon extends Soldier {
+  receiveDamage(damage) {
+    this.health -= damage;
+    if (this.health > 0) {
+      return `A Saxon has received ${damage} points of damage`;
+    } else if (this.health <= 0) {
+      return "A Saxon has died in combat";
+    }
+  }
+}
 
 // War
-class War {}
+class War {
+constructor(vikingArmy, saxonArmy){
+  this.vikingArmy = vikingArmy
+  this.saxonArmy = saxonArmy
+}
```

Comment:
When we work with tests, make sure to read errors! Very helpful. For example, your first two errors for war class:
- should receive 0 arguments
- should assign an empty array to the vikingArmy property

these suggest following: 

```
constructor(){
  this.vikingArmy = []
  this.saxonArmy = []
}
```